app/src/main/java/com/kickstarter/viewmodels/ResetPasswordViewModel.kt

```diff
@@ -127,9 +129,10 @@ interface ResetPasswordViewModel {
         override fun resetError(): Observable<String> {
             return this.resetError
                     .takeUntil(this.resetSuccess)
-                    .map { it.errorMessage() }
+                    .map {  it?.errorMessage() ?: ERROR_GENERIC } /*  null pointer exception is thrown */
+                    //.map { "bad request" }
```

Comment:
Just delete this comment and the comment on the previous line and it is god to go :) 👍